packages/payment-detection/src/thegraph/queries/GetSyncedBlock.graphql

```diff
@@ -0,0 +1,10 @@
+query GetSyncedBlock(
+  $blockHash: Bytes!
+) {
+  _meta(block: { hash: $blockHash }) {
```

Comment:
This is a [time-travel query](https://thegraph.com/docs/developer/graphql-api#time-travel-queries) made to get the subgraph state at a specific block hash.